src/aura/HH_Container/HH_Container.cmp

```diff
@@ -195,6 +207,53 @@
             </aura:if>
             <!-- /DELETE CONTACT POPUP -->
                 
+            <!-- MERGE HH POPUP -->
+            <aura:if isTrue="{!v.showMergeHHPopup}">
+                <div id="mergeHHPopup" class="slds-modal slds-fade-in-open" aria-hidden="false" role="dialog">
+                    <div class="slds-modal__container">
+
+                        <div class="slds-modal__header">
+                            <ui:button class="slds-button slds-button--icon-inverse slds-modal__close" press="{!c.cancelMergeHH}">
+                                <c:svg class="slds-button__icon slds-button__icon--large"
+                                       xlinkHref="/resource/SLDS/1_0_3/assets/icons/action-sprite/svg/symbols.svg#close" />
+                                <span class="slds-assistive-text">{!$Label.c.bdiBtnClose}{!$Label.npsp.bdiBtnClose}</span>
+                            </ui:button>                    
+                            <h2 id="searchPopupLabel" class="slds-text-heading--medium">{!$Label.c.lblMergeHHTitle}{!$Label.npsp.lblMergeHHTitle}</h2>
+                        </div>
+
+                        <div class="slds-modal__content slds-p-around--medium">
+                            <div>
+                                <p>{!format($Label.c.lblMergeHHPrompt, v.conAdd.FirstName + ' ' + v.conAdd.LastName)}
+                                    {!format($Label.npsp.lblMergeHHPrompt, v.conAdd.FirstName + ' ' + v.conAdd.LastName)}</p>
+                            </div>
+                        </div> <!-- /slds-modal__content -->
+                        <div class="slds-modal__footer">
+                            <div class="slds-x-small-buttons--horizontal">
+                                <ui:button class="slds-button slds-button--neutral" 
+                                    aura:id="btnCancelHHMerge"
+                                    buttonTitle="{!$Label.c.stgBtnCancel + $Label.npsp.stgBtnCancel}" 
+                                    label="{!$Label.c.stgBtnCancel + $Label.npsp.stgBtnCancel}" 
+                                    press="{!c.cancelMergeHHPopup}" />
+                                <ui:button class="slds-button slds-button--neutral" 
+                                    aura:id="btnAddContact"
+                                    buttonTitle="{!format($Label.c.lblBtnAddContact, v.conAdd.FirstName + ' ' + v.conAdd.LastName) + 
+                                                 format($Label.npsp.lblBtnAddContact, v.conAdd.FirstName + ' ' + v.conAdd.LastName)}" 
```

Comment:
I can't find `lblBtnAddContact` in the metadata or referenced in `HH_Container_LCTRL`.  Can you check to see if this label is being packaged and also double check the other labels?